testing/camunda-process-test-java/src/main/java/io/camunda/process/test/api/BpmnAssert.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.process.test.api;
+
+import io.camunda.process.test.api.assertions.ProcessInstanceAssert;
+import io.camunda.process.test.impl.assertions.CamundaDataSource;
+import io.camunda.process.test.impl.assertions.ProcessInstanceAssertj;
+import io.camunda.zeebe.client.api.response.ProcessInstanceEvent;
+import io.camunda.zeebe.client.api.response.ProcessInstanceResult;
+import java.time.Duration;
+import org.awaitility.Awaitility;
+
+public class BpmnAssert {
+
+  public static final Duration DEFAULT_ASSERTION_TIMEOUT = Duration.ofSeconds(10);
+  public static final Duration DEFAULT_ASSERTION_INTERVAL = Duration.ofMillis(100);
+
+  private static final ThreadLocal<CamundaDataSource> DATA_SOURCE = new ThreadLocal<>();
```

Comment:
This may also remove the need for the initializing the data source. However, I can imagine it'd be nice to keep it as this will allow users to configure the datasource with their specific runtime.

Reply:
I'm not sure about this. ZPT and the C7 assert use a ThreadLocale. It might be good to use a ThreadLocal because it is a static class. 

If you would run tests in parallel, we would need a ThreadLocal to interact with the correct runtime.